scripts/winesapos-install.sh

```diff
@@ -787,6 +789,18 @@ elif [[ "${WINESAPOS_DE}" == "gnome" ]]; then
         pacman_install_chroot manjaro-gnome-settings manjaro-settings-manager
     fi
     echo "Installing the GNOME desktop environment complete."
+elif [[ "${WINESAPOS_DE}" == "i3" ]]
+      echo "Installing i3"
+    pacman_install_chroot i3-wm xorg xorg-server i3lock i3blocks i3status xcb-util-keysyms xcb-util-cursor xcb-util-wm xcb-util-xrm
```

Comment:
You can install Xorg dependencies here instead:

https://github.com/LukeShortCloud/winesapOS/blob/e97e3c5072ff19c4e89d4db59e73500e30414cbe/scripts/winesapos-install.sh#L734

Reply:
Thanks